libs/components/src/lib/side-drawer/side-drawer.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { elementUpdated, fixture } from '@vivid-nx/shared';
+import { POSITION, SideDrawer } from './side-drawer';
+import '.';
+
+const COMPONENT_TAG = 'vwc-side-drawer';
+
+describe('vwc-side-drawer', () => {
+	let element: SideDrawer;
+
+	beforeEach(async () => {
+		element = await fixture(`<${COMPONENT_TAG}>
+								</${COMPONENT_TAG}>`) as SideDrawer;
+	});
+
+	describe('basic', () => {
+		it('initializes as a vwc-side-drawer', async () => {
+			expect(element).toBeInstanceOf(SideDrawer);
+			expect(element.open).toBeFalsy();
+			expect(element.alternate).toBeFalsy();
+			expect(element.hasTopBar).toBeFalsy();
+			expect(element.position).toBeUndefined();
+			expect(element.modal).toBeFalsy();
+		});
+	});
+
+	describe('show', () => {
+		it('should set "open" to true', async () => {
+			const control = getControlElement();
+			expect(control.classList.toString()).toEqual('control');
+			element.show();
+			await elementUpdated(element);
+			expect(element.open).toEqual(true);
+			expect(control.classList.toString()).toEqual('control open');
+		});
+	});
+
+	describe('hide', () => {
+		it('should set "open" to false', async () => {
+			element.open = true;
+			await elementUpdated(element);
+			const control = getControlElement();
+			expect(control.classList.toString()).toEqual('control open');
+			element.hide();
+			await elementUpdated(element);
+			expect(element.open).toEqual(false);
+			expect(control.classList.toString()).toEqual('control');
+		});
+	});
+
+	describe('modal', () => {
+		it('should set "modal" to true', async () => {
```

Comment:
You actually test two things here.
```suggestion
		it('should set "modal" to true and add "modal" class', async () => {
```